tests/CMakeLists.txt

```diff
@@ -4,3 +4,12 @@ add_subdirectory(simple_executable)
 
 # now include actual tests
 add_subdirectory(core)
+
+add_test(NAME build_minial_appimage
+    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/scripts/build_minimal_appimage_with_custom_apprun.sh
```

Comment:
Way too complex and ambitions, way too fragile, leaves traces on the system, creates a lot of unnecessary I/O, ...

Nothing I would want in the test code.

You must decide whether to do *unit* tests or *integration tests*. Your test fits more into the latter category, but not quite either... We should prefer unit tests, which run a lot faster and are a lot easier to write.

Reply:
This workflow is also covered by a unit test. So it's safe to drop it at this point.